src/components/text-editor/examples/text-editor-composite.tsx

```diff
@@ -33,10 +36,15 @@ export class TextEditorCompositeExample {
                 value={this.value}
                 onChange={this.handleChange}
                 readonly={this.readonly}
+                required={this.required}
                 invalid={this.invalid}
                 placeholder={this.placeholder}
             />,
-            <limel-example-controls>
+            <limel-example-controls
+                style={{
+                    '--example-controls-column-layout': '4',
```

Comment:
this is `4` because when we add the `disabled` prop, there will be 4 checkboxes here